prototypes/basic.py

```diff
@@ -20,7 +19,7 @@ class BasicApplication(object):
 	adminInfo = None
 	accessRights = None
 
-	def __init__(self, moduleName, modulePath, *args, **kwargs):
```

Comment:
I'm quite unsure if its good to remove *args and **kwargs here, because enhanced classes may use extra information and this is the fallback for it, serving as a general function signature.

Reply:
this might also be useful for monkey patching ;-)

Reply:
IMO it's much clearer to have an signature of a method which is reduced on the essentials. To allow kwargs could assume that something is done with them. But this is not the case, so we should omit them.
If you inherit one of these classes you still can change the signature according to your requirements.